xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -1162,17 +1162,32 @@ Use <xref:System.Text.Json.Serialization.ReferenceHandler.Preserve> to enable un
         <summary>Gets or sets a value that indicates whether nullability annotations should be respected during serialization and deserialization.</summary>
         <value>To be added.</value>
         <remarks>
-          <para>Nullability annotations are resolved from the properties, fields and constructor parameters</para>
-          <para>that are used by the serializer. This includes annotations stemming from attributes such as</para>
           <para>
-            <see cref="T:System.Diagnostics.CodeAnalysis.NotNullAttribute" />, <see cref="T:System.Diagnostics.CodeAnalysis.MaybeNullAttribute" />,</para>
+            Configures the serializer to throw an exception when trying to serialize a <see langword="null" /> value from a non-nullable property getter,
+            or when deserializing a <see langword="null" /> value into a non-nullable property setter or constructor parameter.
+            Nullability annotations are resolved from the properties, fields, and constructor parameters
+            that are used by the serializer. This includes annotations stemming from attributes such as
+            <see cref="T:System.Diagnostics.CodeAnalysis.NotNullAttribute" />, <see cref="T:System.Diagnostics.CodeAnalysis.MaybeNullAttribute" />,
+            <see cref="T:System.Diagnostics.CodeAnalysis.AllowNullAttribute" /> and <see cref="T:System.Diagnostics.CodeAnalysis.DisallowNullAttribute" />.
+          </para>
           <para>
-            <see cref="T:System.Diagnostics.CodeAnalysis.AllowNullAttribute" /> and <see cref="T:System.Diagnostics.CodeAnalysis.DisallowNullAttribute" />.</para>
-          <para>Due to restrictions in how nullable reference types are represented at run time,</para>
-          <para>this setting only governs nullability annotations of non-generic properties and fields.</para>
-          <para>It cannot be used to enforce nullability annotations of root-level types or generic parameters.</para>
-          <para>The default setting for this property can be toggled application-wide using the</para>
-          <para>"System.Text.Json.Serialization.RespectNullableAnnotationsDefault" feature switch.</para>
+            Due to restrictions in how nullable reference types are represented at run time,
+            this setting only governs nullability annotations of non-generic properties, fields, and constructor parameters.
+            It cannot be used to enforce nullability annotations of root-level types, collection elements, or generic parameters.
+            The default setting for this property can be toggled application-wide using the
+            "System.Text.Json.Serialization.RespectNullableAnnotationsDefault" feature switch.
+          </para>
+          <para>
+            Nullability validation functions independently of required-ness validation, 
+            that is to say the setting has no effect in scenaria where the payload is missing a required property.
```

Comment:
Isn't scenarios the correct work in English?
```suggestion
            that is to say the setting has no effect in scenarios where the payload is missing a required property.
```

Reply:
Ah, interesting! Well, American English is what we default to in dotnet-api-docs.

Reply:
J/k. Scenari**os** is the prevailing spelling everywhere whereas scenari**a** is a more archaic rendering that I prefer.